worker-kv/tests/integration.rs

```diff
@@ -0,0 +1,103 @@
+use std::{
```

Comment:
If you have bandwidth, could you try getting these tests working with Actions in this repo? It can be a separate workflow for now. 

I'm not sure I understand where / how the `CF_API_TOKEN` is used, but I think we should be able to remove dependencies on a real account and just use Miniflare.

Reply:
All tests in examples/ or just the worker-kv one? The included worker-kv test has some sort of issue, something to do with bad references to `nix`. I am working on figuring out what exactly it is. [Here is the log](https://pastebin.com/XBELw6Dv).

Reply:
Not sure what you mean, I think the `workers-kv` test should run in Actions alongside the other tests. 

Unfortunately I have not seen that issue before.

Reply:
Ah, it turns out that building on Windows has some issues. I resolved it on MacOS. What sort of environment does the Cloudflare team use? WSL? MacOS?

Reply:
> Not sure what you mean, I think the `workers-kv` test should run in Actions alongside the other tests.

Do any of the other tests already included in Actions use Miniflare? Because if not, I will likely have to make a separate Workflow. Workers-kv requires miniflare which requires an `npm install` to grab the proper dependencies.

Reply:
I use MacOS, I'm not sure what others use.